sui-execution/latest/sui-move-natives/src/test_scenario.rs

```diff
@@ -62,10 +106,17 @@ pub fn end_transaction(
     // if true, we will "abort"
     let mut incorrect_shared_or_imm_handling = false;
 
+    let received = object_runtime_ref
+        .test_inventories
+        .allocated_tickets
+        .iter()
+        .map(|(k, (metadata, _))| (*k, metadata.clone()))
+        .collect();
+
     let object_runtime_state = object_runtime_ref.take_state();
     // Determine writes and deletes
     // We pass an empty map as we do not expose dynamic field objects in the system
```

Comment:
Does this comment need updating, or is it related to the other empty map?

Reply:
Will update!